TTBot/DataAccess/Championships.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Dapper;
+using ServiceStack.Data;
+using ServiceStack.OrmLite;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TTBot.Models;
+using TTBot.Services;
+
+namespace TTBot.DataAccess
+{
+    public class Championships : IChampionships
+    {
+        private readonly IDbConnectionFactory _conFactory;
+
+        public Championships(IDbConnectionFactory conFactory)
+        {
+            _conFactory = conFactory;
+        }
+
+        public async Task<int> Exists(string championship)
+        {
+            using (var connection = _conFactory.Open())
+            {
+                return connection.QueryAsync<int>(
```

Comment:
I would paramertise this query or use the method syntax to build it as currently this could be an injection vulnerability
 
You can see examples of OrmLite typed queries here if you're unfamiliar
https://gist.cafe/87164fa870ac7503b43333d1d275456c?docs=8a70f8bf2755f0a755afeca6b2a5238e#doc=select-examples.md&gist=43a09ba142f24ab2ec878284c8b942ca